### PR TITLE
Feat/season spec bb

### PR DESCRIPTION
**Pull request by billyburch**
Great catch on amending the included data files.  We should now have access to the necessary data for our Season methods and tests.

### CODE REVIEW

File: lib/season.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Season
-  def initialize(game_team_data)
+  def initialize(game_team_data, team_data)
```
Comment (EricBelongea) [inline lib/season.rb:2]: Yay, we'll need both data sources as we discussed. Thank you for driving.

**Review by zkevkev — APPROVED** [review]
Thank you for updating this spec! We should be good to go on building methods in this class now.

File: spec/season_spec.rb [hunk 2]
```diff
@@ -16,40 +16,40 @@
     }
 
     @stat_tracker = StatTracker.from_csv(locations)
-    # should we instantiate an object?
+    @season = Season.new(@stat_tracker.game_team_data, @stat_tracker.team_data)
```
Comment (EricBelongea) [inline spec/season_spec.rb:19]: Matches class parameters; Good!

**Review by EricBelongea — APPROVED** [review]
Thank you doing this Billy!!

**Pull request merged**